hawkular-alerts-bus-api/src/main/java/org/hawkular/alerts/bus/api/MetricDataMessage.java

```diff
@@ -86,6 +86,8 @@ public String toString() {
      */

```

Comment:
I think what would be best would be to bypass commons and just have metrics make it public.